Allphi/Domain/Domaincontroller.cs

```diff
@@ -73,17 +74,127 @@ public ParkingSpot GetAvailableParkingSpotVisitor()
             return (_parkingRepo.GetAvailableParkingSpotUnreserved());
         }
 
-        #endregion Parking
-
-        public void SubmitVisitor(string licensePlate)
+        public void SubmitVisitorParking(string licensePlate)
         {
             string license = Guard.Against.NullOrEmpty(licensePlate, nameof(licensePlate));
-            ParkingSpot parkingSpot = GetAvailableParkingSpotVisitor();
+            ParkingSpot parkingSpot = _parkingRepo.GetAvailableParkingSpotUnreserved();
```

Comment:
What's the difference between available and unreserved? Can they be combined to make the naming a bit clearer?